### ISSUE TITLE
Servers are offline or blocked on local host

### ISSUE THREAD

**Issue by fishonamos** [codergautam/swordbattle.io#45]
All servers are offline or blocked on the locally hosted version.

**Comment by codergautam** [comment]
Yes, this is a known bug. I'll work on it sometime. The actual game itself should work fine, it just displays that alert at the beginning.

**Comment by fishonamos** [comment]
Alright. Nice!
